Rmd/Sample_QC.Rmd

```diff
@@ -1278,14 +1278,15 @@ Click on the tabs below to view the plate layout for the experiment (if any was
 
 ```{r is-there-plate-info, echo = F, warning = F}
 are_there_plates <- any(grepl("column", colnames(QAQC_annotated), ignore.case = T))
-```
 
-```{r plate_pos, eval = are_there_plates}
 if (!is.na(params$batch_var)) {
   batch_length <- length(unique(QAQC_annotated[, params$batch_var]))
 } else{
   batch_length <- 1
 }
+```
+
```

Comment:
Interesting. What prompted this?

Reply:
This is actually already in main, I just had to move it into a different chunk than before to prevent an error for data sets that don't have a plate. It's to control the size of the plate figure in the QC report (PR #249 ).